playbooks/adhoc/uninstall.yml

```diff
@@ -21,6 +21,7 @@
 
     - set_fact:
         is_atomic: "{{ ostree_output.rc == 0 }}"
+        is_containerized: is_atomic or containerized | default(False)
```

Comment:
These should probably be 2 different facts..  Ansible will throw a big blue warning if trying to use set_fact variables that reference set_fact variables from the same task.

It's always *seemed* to work when I've done it, but I wouldn't trust it.